src/status_im/utils/notifications.cljs

```diff
@@ -20,7 +20,7 @@
   (.requestPermissions (.-default rn/react-native-fcm)))
 
 (defn get-fcm-token []
-    (-> (.getFCMToken (aget rn/react-native-fcm "default"))
+    (-> (.getFCMToken (.-default rn/react-native-fcm))
```

Comment:
this one is very consistent with the one above.

Reply:
@poiga is there a reason you're mixing object/get and direct property access?  (even for non-hyphened properties). 95% of the time you're using object/get, but in some places like this one direct property call.

Reply:
@dmitryn can you see on the line 20 in the same file is the exact string - so I just wanted to keep it consistent  and readable.

Reply:
@poiga afaik, there are tradeoffs of using `(goog.object/get obj "prop")` or `(.-prop obj)`. Latter can be optimized by cljs compiler to reduce bundle size and obfuscate code in `:advanced` mode, but externing would be required, more info here https://github.com/gphilipp/react-native-with-clojurescript#building

We don't care much about obfuscating as we're open-source. Not sure about bundle size though, reducing its size might get us some benefits.

Overall, i think we should use dot notation everywhere except hyphened properties (where goog.object/get could be used).